### ISSUE TITLE
DRAFT: Support various access levels for embedded map features

### ISSUE THREAD

**Issue by dboyer** [open-apparel-registry/open-apparel-registry#1292]
# Draft

This is a draft issue, and it will need to clarify specs and formalize what
counts as "done" before work begins on it.

Check with the issue's creator before starting work on this issue.

## Overview

Access to embedded map functionality will be available at three different levels that will allow access to different functionality.

### Describe the solution you'd like

Exact details are still in discussion but the different functionality will be:

- Set number of pre-set and custom data fields at each of three levels
- Access to set custom colors at some of the three levels
- Volume of facilities on embedded map

### Additional Context

Optional. Add any other context, screenshots, gifs, etc about the feature request here.

**Comment by dboyer** [comment]
Duplicate of #1335
